BackEndApp/v1/views.py

```diff
@@ -1,3 +1,4 @@
+from genericpath import isdir
```

Comment:
Can we remove this?

Reply:
Yes, I believe we can... This was present before I added any changes myself :P
There are many such imports which aren't being used as well... should I remove those as well??

Reply:
Well I'd suggest removing only the ones you added for this PR
You can most definitely make a cleaning PR to remove others (Or make a good first issue for new contributors :P )

Reply:
Ahh sure I'll make an issue for new contributors, that seems like the better option :D